docs/source/usage/quick_start.rst

```diff
@@ -65,21 +79,46 @@ With the DataFrame loaded, you can now generate a :obj:`MetaFrame <metasyn.metaf
 This creates a MetaFrame named ``mf``.
 
 .. Note:: 
-	At this point, you might get a warning about a potential unique variable, but we can ignore that for now as it's safe to continue.
-	
-	``Variable PassengerId seems unique, but not set to be unique. Set the variable to be either unique or not unique to remove this warning. warnings.warn(f"\nVariable {series.name} seems unique, but not set to be unique.\n"``
+	At this point you will get a warning because metasyn detects that 'PassengerId' contains unique values, but was not specified by us to generate only unique values. The warning is as follows:
```

Comment:
```suggestion
	At this point you will get a warning because metasyn detects that 'PassengerId' contains unique values, but we did not specify this column to generate only unique values. The warning is as follows:
```